vmware_aria_operations_integration_sdk/project.py

```diff
@@ -199,3 +235,20 @@ def _add_and_update_project_paths(path: str) -> None:
             )
         ),
     )
+
+
+def _safe_append_to_gitignore(gitignore_file_path: str, token: str) -> None:
+    try:
+        with open(gitignore_file_path, "r") as gititnore:
+            for line in gititnore.readlines():
+                if token in line:
```

Comment:
We probably want to match the whole line, rather than just (potentially) a part of it.